components/UserSaysProperty.js

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react'
+
+const UserSaysProperty = ({ panel, onSaveUserSaysPropertiesClick, onUserSaysChange }) => (
```

Comment:
This component is almost the same as ResponseProperty. Is it possible that they use the same component but with different properties? If you don't have time to refactor it right now, we can do it after the deadline of this project.